INSTALL.md

```diff
@@ -69,40 +69,36 @@ brew info elasticsearch
 ```
 
 ### MYSQL Database
-Ensure your database is running.
+Ensure your database is running with the following command:
 ```
 mysql.server start
 ```
 
-Then run mysql creation script from root directory:
+Then run the MYSQL creation script from project root directory:
 ```
 ./create-mysql-db.sh
 ```
-If you would like to have a custom db setup then you can pass in the necessary args.
-> **NOTE** be sure to update your local settings `local.py` to account for these changes.
+
+If you would like to have a custom database setup then you can pass in the necessary arguments:
 
 ```
 ./create-mysql-db.sh <dbname> <username> <password>
 ```
 
+> **NOTE:** Be sure to update your local settings in
+  `cfgov/cfgov/settings/local.py` to account for these changes.
 
 
-### Sheer-like & Sheer
-> **NOTE** Ensure you install Sheer-like first to avoid version conflicts with dependencies.
-
-To [install Sheer-like](https://github.com/cfpb/django-sheerlike/blob/master/README.rst), start by cloning the
-Sheer GitHub project to wherever you keep your projects (not inside cfgov-refresh directory):
+### Sheer-like
+To [install Sheer-like](https://github.com/cfpb/django-sheerlike/blob/master/README.rst),
+start by cloning the Sheer-like GitHub project to wherever you keep your projects
+(not inside cfgov-refresh directory):
 ```bash
 git clone https://github.com/cfpb/django-sheerlike.git
 ```
 
-To [install Sheer](https://github.com/cfpb/sheer#installation), start by cloning the
-Sheer GitHub project to wherever you keep your projects (not inside cfgov-refresh directory):
-```bash
-git clone https://github.com/cfpb/sheer.git
-```
-
-Go back to the cfgov-refresh directory and workon a virtualenv, which you’ll name `cfgov-refresh`:
+Go back to the cfgov-refresh directory and workon a virtualenv,
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/5d06e4dbdbc421dc9f5c6b64773357caedabcff9